frontend/packages/component-library/src/button/_BaseButton.tsx

```diff
@@ -269,6 +274,12 @@ const BaseButton: React.FunctionComponent<_BaseButtonProps> = ({
         moduleStyles[`button-${type}`],
         moduleStyles[`button-${color}`],
         moduleStyles[`button-${size}`],
+        {
```

Comment:
this seems too complicated too much classnames for just showing hover state of the button